packages/web-app-files/src/mixins/spaces/actions/delete.js

```diff
@@ -10,9 +10,13 @@ export default {
           label: () => {
             return this.$gettext('Delete')
           },
-          handler: this.$_delete_showModal,
-          isEnabled: (space) => {
-            return space.root?.deleted?.state === 'trashed'
+          handler: this.$_delete_trigger,
+          isEnabled: ({ spaces }) => {
+            if (spaces.length === 0) {
```

Comment:
you can check for `spaces.length !== 1` here since you only allow the action for a single space as of now.